### PR TITLE
site: icon update

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | site: icon update |
| 🇨🇳 Chinese | site: icon update |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/common/ThemeSwitch/index.tsx [hunk 1]
```diff
@@ -11,41 +12,43 @@ export type ThemeSwitchProps = {
...
-);
+const ThemeSwitch: React.FC<ThemeSwitchProps> = ({ value, onChange }) => {
+  const navigate = useNavigate();
+  return (
+    <FloatButton.Group trigger="click" icon={<ThemeIcon />}>
+      <FloatButton
+        icon={<BgColorsOutlined />}
+        tooltip={<FormattedMessage id="app.footer.theme" />}
+        onClick={() => {
+          navigate('/theme-editor');
```
Comment (MadCcc) [inline .dumi/theme/common/ThemeSwitch/index.tsx:23]: 用 href

Comment (li-jia-nan) [reply .dumi/theme/common/ThemeSwitch/index.tsx]: done

Comment (li-jia-nan) [reply .dumi/theme/common/ThemeSwitch/index.tsx]: 用 Link 组件有个不好的地方就是需要手动加上 style，不然就丢掉 margin 了

File: .dumi/theme/common/ThemeSwitch/index.tsx [hunk 2]
```diff
@@ -13,21 +14,19 @@ export type ThemeSwitchProps = {
...
-      icon={<Light />}
-      type={!value.includes('dark') ? 'primary' : 'default'}
-      onClick={() => {
-        if (value.includes('dark')) {
-          onChange(value.filter((theme) => theme !== 'dark'));
-        }
-      }}
-      tooltip={<FormattedMessage id="app.theme.switch.default" />}
-    />
+    <Link to="/theme-editor" style={{ display: 'block', marginBottom: 16 }}>
```
Comment (MadCcc) [inline .dumi/theme/common/ThemeSwitch/index.tsx:17]: 链接要国际化一下，有个工具方法可以用

**Comment by MadCcc** [comment]
image
紧凑模式下间距不对了

File: .dumi/theme/common/ThemeSwitch/index.tsx (hunk 2 above)
Comment (li-jia-nan) [reply .dumi/theme/common/ThemeSwitch/index.tsx]: done

File: .dumi/theme/common/ThemeSwitch/index.tsx [hunk 3]
```diff
@@ -11,41 +13,45 @@ export type ThemeSwitchProps = {
...
-    />
-  </FloatButton.Group>
-);
+const ThemeSwitch: React.FC<ThemeSwitchProps> = ({ value, onChange }) => {
+  const { pathname, search } = useLocation();
+  return (
+    <FloatButton.Group trigger="click" icon={<ThemeIcon />}>
+      <Link
+        to={getLocalizedPathname('/theme-editor', isZhCN(pathname), search)}
+        style={{ marginBottom: value.includes('compact') ? 8 : 16, display: 'block' }}
```
Comment (MadCcc) [inline .dumi/theme/common/ThemeSwitch/index.tsx:22]: 直接用 token 吧

Comment (li-jia-nan) [reply .dumi/theme/common/ThemeSwitch/index.tsx]: done

File: .dumi/theme/common/ThemeSwitch/index.tsx [hunk 4]
```diff
@@ -11,41 +14,49 @@ export type ThemeSwitchProps = {
...
-);
+const ThemeSwitch: React.FC<ThemeSwitchProps> = ({ value, onChange }) => {
+  const { token } = useSiteToken();
+  const { pathname, search } = useLocation();
+  return (
+    <FloatButton.Group trigger="click" icon={<ThemeIcon />}>
+      <Link
+        to={getLocalizedPathname('/theme-editor', isZhCN(pathname), search)}
+        style={{
+          marginBottom: value.includes('compact') ? token.marginXS : token.margin,
```
Comment (MadCcc) [inline .dumi/theme/common/ThemeSwitch/index.tsx:25]:
```suggestion
          marginBottom: token.margin,
```
会自动计算的

Comment (li-jia-nan) [reply .dumi/theme/common/ThemeSwitch/index.tsx:25]: 厉害了

**Review by MadCcc — APPROVED** [review]

**Pull request merged**